storybook/main.js

```diff
@@ -63,6 +64,9 @@ module.exports = {
 	docs: {
 		autodocs: true,
 	},
+	typescript: {
+		reactDocgen: 'react-docgen-typescript',
+	},
```

Comment:
Needed since the default in v8 is `react-docgen`, see: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#react-docgen-component-analysis-by-default